Makefile

```diff
@@ -426,23 +426,24 @@ test-size-solver : install-size-solver
 # Agda can fail to compile on Windows if files which are CPP-processed
 # don't end with a newline character (because we use -Werror).
 
-FAW_PATH = src/fix-agda-whitespace
-FAW_BIN  = $(FAW_PATH)/dist/build/fix-agda-whitespace/fix-agda-whitespace
+FAW_PATH = src/fix-whitespace
```

Comment:
Maybe since it is no longer called `fix-agda-whitespace` the variable prefix shouldn't be `FAW` but e.g. `FIXW`.

Reply:
Sounds good to me, so they are all renamed.